roles/spark/templates/build.gradle.j2

```diff
@@ -0,0 +1,28 @@
+/* * This build file was generated by the Gradle 'init' task.
```

Comment:
No templaty things used in file, move it to roles/spark/files/build.gradle.
Or make netlib version a variable.